website/project/model.py

```diff
@@ -860,6 +860,7 @@ class Node(GuidStoredObject, AddonModelMixin, IdentifierMixin):
     primary_institution = fields.ForeignField('institution')
 
     affiliated_institutions = fields.ForeignField('institution', list=True)
+    visit = fields.IntegerField(default=0)
```

Comment:
You could still calculate submissions by tags, but this would also (going forward) tell you whether or not a give node was ever connected to a conference, and would be helpful in fixing submissions with their tags erroneously removed.